Command/CheckConfigCommand.php

```diff
@@ -3,6 +3,7 @@
 namespace Lexik\Bundle\JWTAuthenticationBundle\Command;
 
 use Lexik\Bundle\JWTAuthenticationBundle\Services\KeyLoader\KeyLoaderInterface;
+use RuntimeException;
```

Comment:
Sorry to ask but can you please revert this change? I prefer the current approach as it makes it easy to distinguish built-in exceptions from domain ones

Reply:
Yes, I also need to find the rules rector to disable this change

Reply:
Done 😉